src/SingleMovie/SingleMovie.test.js

```diff
@@ -1,28 +1,76 @@
-    // we need to render SingleMovie, making sure to include the movieDetails and returnHome in the render
-    // execution
-    // 
-    // assertion
-    // assert that all the movie details and existing inner text display with 
-    // use of expect toBeInTheDocuments
-  })
+  it('should render details for a single movie', () => {
+    const movie = {
+      "id": 694919,
+      "title": "Money Plane",
+      "poster_path": "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+      "release_date": "2020-09-29",
+      "overview": "A professional thief with $40 million in debt and his family's life on the line must commit one final heist - rob a futuristic airborne casino filled with the world's most dangerous criminals.",
+      "genres": [
+        "Action"
+      ],
+      "budget": 0,
+      "revenue": 0,
+      "runtime": 82,
+      "tagline": "",
+      "average_rating": 6.666666666666667
+    }
+  
+   const mockReturnToHome = jest.fn();
+    
+    render(
+      <SingleMovie
+        movieDetails={movie}
+        returnToHome={mockReturnToHome}
+      />
+    )
 
-  // Integration test for app
-  it('should take user to movie cards view when return to home button is clicked', () => {
-     // setup
-     // render single movie with movieDetails and mocked returntohome
-     // execution
-     // fireEvent or userEvent for onclick of Return to home button
-     // assertion
-     // expect movieContainer to show cards
-  })
+    const image = screen.getByRole('img', { name: /money plane/i });
+    const title = screen.getByRole('heading', { name: /money plane/i });
+    const overview = screen.getByRole('heading', { name: /overview: a professional thief with \$40 million in debt and his family's life on the line must commit one final heist \- rob a futuristic airborne casino filled with the world's most dangerous criminals\./i });
+    expect(image).toBeInTheDocument();
+    expect(title).toBeInTheDocument();
+    expect(overview).toBeInTheDocument();
+  }) 
+  
+  it('should invoke return to home method when Return To Home button is clicked', () => {
+    const movie = {
+      "id": 694919,
+      "title": "Money Plane",
+      "poster_path": "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+      "backdrop_path": "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+      "release_date": "2020-09-29",
+      "overview": "A professional thief with $40 million in debt and his family's life on the line must commit one final heist - rob a futuristic airborne casino filled with the world's most dangerous criminals.",
+      "genres": [
+        "Action"
+      ],
+      "budget": 0,
+      "revenue": 0,
+      "runtime": 82,
+      "tagline": "",
+      "average_rating": 6.666666666666667
+    }
+  
+    const mockReturnToHome = jest.fn();
+
+    render(
```

Comment:
I wonder if we need to re-work this just a bit, as I believe we want render the movie container when returnToHome is invoked? I'm not sure though.